src/command_modules/azure-cli-container/azure/cli/command_modules/container/custom.py

```diff
@@ -345,8 +345,8 @@ def add_ci(
 
     # Call the RP
     return _call_rp_configure_cicd(
-        name,
-        resource_group_name,
+        target_name,
+        target_resource_group,
```

Comment:
I don't understand the reason for these renames. `name` and `resource_group_name` are consistent with the other modules in the CLI and, as far as I can tell, they function the same in these commands as they do in those modules.